src/cmd/generate.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/nasz-elektryk/spito/checker"
+	"github.com/nasz-elektryk/spito/cmd/cmdApi"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+const exampleRuleContents = `function main()
+  return true
+end
+`
+
+func handleGenerate(cmd *cobra.Command, args []string) {
+	configFileContents, error := os.ReadFile(checker.CONFIG_FILENAME)
+	if os.IsNotExist(error) {
+		printErrorAndExit(errors.New("Please run this commnd inside an actual spito ruleset directory!"))
+	}
+	handleError(error)
+
+	config := ConfigFileLayout{}
+	error = yaml.Unmarshal(configFileContents, &config)
+	handleError(error)
+
+	rulesetLocation := checker.RuleSetLocation{}
+	rulesetLocation.New(config.Identifier)
+	if !rulesetLocation.IsRuleSetDownloaded() {
```

Comment:
I don't understand this line
When someone is downloading a new ruleset it does makes sens that it needs to be putted into specific directory.
But if someone just wants to create a new rule in ruleset it should look like a normal project, meaning that it should be allowed to create ruleset wherever you want to.